src/OrchardCore/OrchardCore.Users.Abstractions/IUser.cs

```diff
@@ -1,9 +1,11 @@
+using OrchardCore.Entities;
+
 namespace OrchardCore.Users
 {
     /// <summary>
     /// Marker interface for ASP.NET Core Identity services.
     /// </summary>
-    public interface IUser
+    public interface IUser : IEntity
```

Comment:
Not sure about it. It'd be better to keep it at the class level. Only the Users module needs to know about this IMO, even if it means casting sometimes.

Reply:
Ah I thought I had kept it to the Users module (the namespace on this class doesn't match the project it is in). I'll change it.